databases/mongodb-azure-cosmosdb/script.ts

```diff
@@ -0,0 +1,48 @@
+import { PrismaClient } from '@prisma/client'
+
+const client = new PrismaClient()
+
+// A `main` function so that we can use async/await
+async function main() {
+  await client.user.deleteMany({})
+
+  const objectId = '6d795f757365725f69643030'
+
+  await client.user.create({
+    data: {
+      id: objectId,
+      email: 'alice@prisma.io',
+      name: 'Alice',
+    },
+  })
```

Comment:
Looks like this create has concurrency issue (binary / library test running at the same time)
https://github.com/prisma/e2e-tests/runs/5219187664?check_suite_focus=true#step:6:153

Though it works in the end

Reply:
Isn't that the same with all our database testing scripts? (deleteMany of `binary` can also just remove these creates of `library` again)

Reply:
It's not amazing but it works with the retries